scripts/area_plot.py

```diff
@@ -0,0 +1,613 @@
+import argparse
+import warnings
+# from warnings import Warning
+
+from matplotlib import rcParams
+from matplotlib import cm
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+"""
+A set of python scripts to generate an area plot.
```

Comment:
This is nitpicky but it's just one script at the moment ;). Anyway, I prefer something more descriptive and straight to the point:
```
"""Generate area plot from taxonomic profiling table.
"""
```